api/MediaError.json

```diff
@@ -52,10 +52,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaError/code",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "43"
```

Comment:
This is an attribute and http://mdn-bcd-collector.appspot.com/tests/api/MediaError/code is only looking at the prototype, so this 43 is when attributes were moved to prototypes. I'm pretty sure this actually goes back to when the interface itself was introduced, which in turn should match when api.HTMLMediaElement.error` was introduced.